dashboard/test/fixtures/gatekeeper_invalid.yaml

```diff
@@ -0,0 +1,35 @@
+# This yaml file describes the configuration for
+# gatekeeper.  Here is how a typical feature would
+# be defined:
+#
+# <feature name>:
+# - rule: <descripion of what rule does>
```

Comment:
Right the rule is only there to make it more logical for humans to define gatekeeper rules. I'll update